content/collective_intro.tex

```diff
@@ -20,8 +20,8 @@
 array if all previous collective routines using the \VAR{pSync} array have been
 completed by all participating \acp{PE}.  One can use a synchronization
 collective routine such as \barrier to ensure completion of previous collective
-routines. The \FUNC{shmem\_barrier} routine allows the same \VAR{pSync} array to
-be used on consecutive calls as long as the \ac{PE} \activeset does not change. 
+routines. The \FUNC{SHMEM\_BARRIER} routine allows the same \VAR{pSync} array to
```

Comment:
The use of stylized capitalization for routines in the collectives intro is not consistent with lower-case use by the rest of the document.